src/tran.rs

```diff
@@ -6459,14 +6459,8 @@ impl<'a> InstructionTranslator<'a> {
             ),
             [
                 self.section.state_ptr,
-                // LLVMBuildBitCast(
-                //     self.builder,
-                //     self.section.state_ptr,
-                //     LLVMPointerType(LLVMInt8Type(), 0),
-                //     NONAME,
-                // ),
                 aligned_addr,
-                LLVMConstInt(LLVMInt8Type(), size as u64, 0),
+                LLVMConstInt(LLVMInt8Type(), 2 as u64, 0),
```

Comment:
I think this deserves a comment to explain why we always load a full word here. Can you please add one?

Reply:
Addressed in [this commit](https://github.com/pulp-platform/banshee/pull/18/commits/95619028583909da22c571e6fdbf3cd400d35545).